crates/revm/revm-inspectors/src/tracing/js/mod.rs

```diff
@@ -324,32 +327,31 @@ where
 
     fn step_end(
         &mut self,
-        interp: &mut Interpreter,
-        data: &mut EVMData<'_, DB>,
-        eval: InstructionResult,
+        _interp: &mut Interpreter<'_>,
+        _data: &mut EVMData<'_, DB>,
     ) -> InstructionResult {
```

Comment:
@rakita why was this removed, or has this been moved to the `Interpreter.instruction_result`?

Reply:
It was the same thing, `interpreter.instruction_result` should be used

Reply:
Nice, that answers my above q - thanks!